src/main.cpp

```diff
@@ -595,13 +688,11 @@ void bleServerInit(){
   // Create a BLE Characteristic for PM 2.5
   pCharactData = pService->createCharacteristic(
       CHARAC_DATA_UUID,
-      BLECharacteristic::PROPERTY_READ | BLECharacteristic::PROPERTY_NOTIFY
```

Comment:
please restore the format code here, on the bluetooth callbacks.

Reply:
done